test/e2e/tests/test_uninstall.py

```
@@ -0,0 +1,332 @@
+    "externalmodels.maas.opendatahub.io",
+    "tenants.maas.opendatahub.io",
+]
+
+MAAS_WORKLOAD_LABELS = [
+    "app=maas-controller",
+    "app=maas-api",
+]
+
+
+def _oc(*args, timeout=60, check=False):
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)